cloud_build/app_flutter_build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright 2020 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Build flutter app to generate flutter build dashboard.
+
+pushd app_flutter > /dev/null
+rm -rf build
+flutter pub get
```

Comment:
Nit: consider adding `flutter doctor` so that it's easier to debug when something goes wrong.

Reply:
Added.